<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MilitaryCommander</class>
 <widget class="QWidget" name="MilitaryCommander">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="commanderList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>80</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>120</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QTextEdit" name="suduText"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>章节</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="colorText"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="wuliText"/>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="zhiliText"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>武力</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="chapterText"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>智力</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>速度</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>模型</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="modelText"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>颜色</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>160</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>简体名字</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>繁体名字</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="traditionalNameText"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="simpliedNameText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>146</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="difangliupaiText"/>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="diaobaoliupaiText"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>我方流派</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>掉宝流派</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="wofangliupaiText"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>敌方流派</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>脸谱</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="faceText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>146</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>脸谱控制</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>繁体控制</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="faceControlText"/>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="chtNameControlText"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>361</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="7" column="0">
     <widget class="QCheckBox" name="skillMing">
      <property name="text">
       <string>命</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="skillJue">
      <property name="text">
       <string>觉</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="skillFan">
      <property name="text">
       <string>返</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="skillWu">
      <property name="text">
       <string>武</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="skillBi">
      <property name="text">
       <string>避</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="skillGong">
      <property name="text">
       <string>攻</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="skillHui">
      <property name="text">
       <string>慧</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="skillZhi">
      <property name="text">
       <string>智</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="skillRen">
      <property name="text">
       <string>仁</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="skillHun">
      <property name="text">
       <string>魂</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="skillDang">
      <property name="text">
       <string>挡</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="skillTong">
      <property name="text">
       <string>统</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QCheckBox" name="skillFen">
      <property name="text">
       <string>奋</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QCheckBox" name="skillShi">
      <property name="text">
       <string>识</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="skillFang">
      <property name="text">
       <string>防</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="skillMou">
      <property name="text">
       <string>谋</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QCheckBox" name="skillYi">
      <property name="text">
       <string>医</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QCheckBox" name="skillLin">
      <property name="text">
       <string>临</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>110</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
